### DISCUSSION TITLE
_cpyfrom keyword

### DISCUSSION THREAD

**Discussion by CSWinnall** [cortexrd/Knack-Toolkit-Library#271] [General]
Hi All

We are trying to use the cpyfrom keyword in our app.  Has anyone used it yet and if they have we would like to see some examples of what you have used it for.

Craig

**Comment by cortexrd** [comment]
Hi Craig,

Yes, I will assist you with this, as it's not trivial.

And BTW... maybe it's just a typo, but it's **_cpyfrom** not _copyfrom in your title.

Norm

**Reply by CSWinnall** [reply]
Fixed it.

I'm trying to find out how people are using it.  I think it is going to be very useful but I can't picture it at the moment so I'm looking for inspiration.

**Comment by CSWinnall** [comment]
Norm

I think I understand how to get the keyword to work sort of.  I just cannot picture uses for it.  Can you describe a problem that you had and how cpyfrom fixes that problem.  Your Wiki describes how to use it, I think sometimes it may be better to describe a use case as well. I know you can't always do that.

Craig

**Comment by Ampurcell** [comment]
Hello, 

I am interested in this keyword as well. But I'm not 100% clear if it is what I am looking for or how to get it to work.
Maybe I am missing something another keyword would be better. 

Basically, I have a list of records I would like to copy at once. 

Example: About 50 products connected to an invoice. I want to copy all the same items to another invoice with minimal clicks.
